.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,22 @@
+name: Python application
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install pynwb
+      run: |
+        pip install git+https://github.com/hdmf-dev/hdmf.git
```

Comment:
I'll change this to the released version once we release the recently added validator running code